### PR TITLE
issue #84: Wallet buttons for receiving and send AKA

### PR DESCRIPTION
**Pull request by adonayvasquezz**
[issue-84](https://github.com/akroma-project/akroma-wallet-mobile/issues/84)

## Description
- Add buttons to the wallet details section for receiving & send AKA
- Buttons in options menu for receiving and send AKA removed

Captura de Pantalla 2022-12-16 a la(s) 10 12 13

### CODE REVIEW

File: src/components/DetailsScreenHeaderRight.tsx [hunk 1]
```diff
@@ -23,21 +23,15 @@ export const DetailsScreenHeaderRight = () => {
       onPress={() =>
         showActionSheetWithOptions(
```
Comment (cafadev) [inline src/components/DetailsScreenHeaderRight.tsx:24]: Can we place this function before the `return` ?. In order to keep the JSX easy to read.

**Review by cafadev — CHANGES_REQUESTED** [review]
Looks great 👍🏼 . Just resolve the small changes.

File: src/components/DetailsScreenHeaderRight.tsx (hunk 1 above)
Comment (adonayvasquezz) [reply src/components/DetailsScreenHeaderRight.tsx:24]: Yes, On it!

**Comment by cafadev** [comment]
@adonayvasquezz help me to resolve the conflicts! Thank you! 👍🏼

**Review by cafadev — APPROVED** [review]

**Pull request merged**